gpg/haydn_gpg_key

```diff
@@ -1,19 +1,16 @@
 -----BEGIN PGP PUBLIC KEY BLOCK-----
 
-mDMEYk4LthYJKwYBBAHaRw8BAQdA/W3MNht2e9OUUTDm5EIakCPuAQoKQ+PCvbPh
-YwK3Da60N0hheWRuIEdyZWF0bmV3cyAoS2V5IGZvciAyMDIyKSA8aGF5ZG5Ac3By
-aW5nbG9hZC5jby5uej6IlgQTFggAPhYhBJL0jmsMsCZxK8fcnqpMqGQ1oVmCBQJi
-Tgu2AhsDBQkCx+oABQsJCAcCBhUKCQgLAgQWAgMBAh4BAheAAAoJEKpMqGQ1oVmC
-6wkA/1zo/sKeya4OQD6SLxTNBAZjB0KmDrpBx8KgVRPR63KjAP9ofWIYPledjJOA
-UrSvaN0ZPlrPBbS86IhOQXXdrrxdA4h1BBAWCAAdFiEEKiZ+q4y7pRTZUoLMDnX9
-0uBEBNAFAmJODDgACgkQDnX90uBEBNDdNAD/U6KS2txNyDG9wTOf+ET3sP8ZGu//
-SAcKLiG0bBxCLm4BANyEeVktzQQ/wdmcXJ42RbVmXqh0EbyDDY0j5WcfKaoCiHUE
-EBYIAB0WIQRMg+VrF/f1qW8wwhWRLuOvRP59DQUCYk4MxwAKCRCRLuOvRP59DUF2
-AQDxKWB1YV4r8CvFiJgLKe+5BRNd3MBvgFuM9SO/fAqaiQD+KTgAB9By33KlsuFM
-xGCD8PoPoebcA4rX5Dy3WJD+/wC4OARiTgu2EgorBgEEAZdVAQUBAQdASQLTMUCv
-DWyd78zH0O7iUh022KD8iu2fubbJN8WP+R0DAQgHiH4EGBYIACYWIQSS9I5rDLAm
-cSvH3J6qTKhkNaFZggUCYk4LtgIbDAUJAsfqAAAKCRCqTKhkNaFZgiqPAP0VYL/3
-XJx+NMrEml4avJpWmKPW/Rz18bm7p9zfZT1ZMwD/TH6awyZCACVjp/5ZwUq7KPG7
-qIIA2e+Yv29qxVfjFAg=
-=0136
+mDMEZRyoPBYJKwYBBAHaRw8BAQdAH1eX4xm1p+T7W72nbAPhe5xmUJ1gc/HV4nCf
```

Comment:
I believe the new format is `me_gpg.key` as per https://handbook.springload.co.nz/tech-team/key-management/gpg/ (not that it matters much)